scholarx/archive/2020/mentees.json

```diff
@@ -0,0 +1,426 @@
+    },
+
+    {
+        "name": "Shehan Munasinghe",
+        "image": "shehan.jpg",
+        "mentor": "Chethiya Abeysinghe",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/shehanmunasinghe/"
+    },
+
+    {
+        "name": "Chiran Hewawitharana",
+        "image": "chiran.jpg",
+        "mentor": "Ganindu Nanayakkara",
+        "university": "University of Kelaniya",
+        "linkedin": "https://www.linkedin.com/in/chiran-hewawitharana/"
+    },
+
+    {
+        "name": "Lahiru Seneviathne",
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Ganindu Nanayakkara",
+        "university": "University of Moratuwa",
+        "linkedin": ""
+    },
+
+    {
+        "name": "Abdur Azeez",
+        "image": "abdur.jpg",
+        "mentor": "Dr. Hasini Jayatilaka",
+        "university": "University of Nottingham",
+        "linkedin": "https://www.linkedin.com/in/recnacara-abdur-rahmaan-azeez-aralebon/"
+    },
+
+    {
+        "name": "Karthika Jeganathan",
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Dr. Hasini Jayatilaka",
+        "university": "Edinburgh Napier University",
+        "linkedin": ""
+    },
+
+    {
+        "name": "Lasni Nishshanke",
+        "image": "lasni.jpg",
+        "mentor": "Iromi Wanigasuriya",
+        "university": "University of Sri Jayewardenepura",
+        "linkedin": "https://www.linkedin.com/in/lasni-nishshanke-9b1612142/"
+    },
+
+    {
+        "name": "Bimsara Pathiraja",
+        "image": "bimsara.jpg",
+        "mentor": "Isuru Daulagala",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/bimsara-pathiraja-2536a913b/"
+    },
+
+    {
+        "name": "Tharuka Sarathchandra",
+        "image": "tharuka.jpg",
+        "mentor": "Isuru Daulagala",
+        "university": "University of Kelaniya",
+        "linkedin": "https://www.linkedin.com/in/tharukav/"
+    },
+
+    {
+        "name": "Chalani Ekanayake",
+        "image": "chalani.jpg",
+        "mentor": "Isuru Daulagala",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/chalaniekanayake/"
+    },
+
+    {
+        "name": "Nadarasar Bahavan",
+        "image": "nadarasar.jpg",
+        "mentor": "Jeewika Ranaweera",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/nadarasar-bahavan/"
+    },
+
+    {
+        "name": "Pathum Goonathilake",
+        "image": "pathum.jpg",
+        "mentor": "Jeewika Ranaweera",
+        "university": "General Sir John Kotelawala Defence University",
+        "linkedin": "https://www.linkedin.com/in/pathum-goonathilake-0181b7140/"
+    },
+
+    {
+        "name": "Muraleetharan Mathanraj",
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Jeewika Ranaweera",
+        "university": "University of Jaffna",
+        "linkedin": ""
```

Comment:
https://www.linkedin.com/in/mathanraj-sharma/ 
Please update the name and the headshot as well.